docs/intermediate-usage.rst

```diff
@@ -132,12 +132,11 @@ exercise a larger amount of options. We'll define a resource named "User". ::
 
         @marshal_with(user_fields)
         def get(self, id):
-            args = get_parser.parse_args()
+            args = post_parser.parse_args()
             user = fetch_user(id)
             return user
 
-As you can see, we create a `post_parser` specifically to handle parsing
-the arguments provided on POST. Let's step through the definition of each
+As you can see, we create a `post_parser` specifically to handle the parsing of arguments provided on POST. Let's step through the definition of each
```

Comment:
Another thing you can fix is that variable names in paragraphs need double backticks to be formatted as `monospaced`